Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -154,19 +155,35 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
     ctxVisible,
     hVisible
   ) => {
-    const visiblity =
+    const visibility =
       renderingType === RenderingTypes.PICKING_BUFFER
         ? wVisible
         : wVisible && hVisible;
 
     publicAPI.getActors().forEach((actor) => {
       actor.getProperty().setOpacity(model.opacity);
+      let actorVisibility = visibility;
+
+      // Conditionally display rotation handles
       if (publicAPI.getRotationActors().includes(actor)) {
-        actor.setVisibility(visiblity && model.rotationEnabled);
-      } else {
-        actor.setVisibility(visiblity);
+        actorVisibility = visibility && model.rotationEnabled;
```

Comment:
`actorVisibility = actorVisibility && model.rotationEnabled;` seems to be more robust (in case a new condition is added before `if( publicAPI.getRotationActors().includes(actor)) `)